### PR TITLE
Add --show-non-errors to jshint [#1179]

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @jbuck received. Current queue size: 0

Live output at: http://butyr.org:8000/fc59cbdf808ab4c/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://butyr.org:8000/fc59cbdf808ab4c/output.txt

Total script time: 0.65 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @jbuck received. Current queue size: 0

Live output at: http://butyr.org:8000/c76e3f450f7b1a5/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://butyr.org:8000/c76e3f450f7b1a5/output.txt

Total script time: 0.24 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @jbuck received. Current queue size: 0

Live output at: http://butyr.org:8000/f96f4cd32b5d105/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://butyr.org:8000/f96f4cd32b5d105/output.txt

Total script time: 0.55 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @jbuck received. Current queue size: 0

Live output at: http://butyr.org:8000/75a4071b91f97e5/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://butyr.org:8000/75a4071b91f97e5/output.txt

Total script time: 0.37 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @jbuck received. Current queue size: 0

Live output at: http://butyr.org:8000/9c9914c2855d0b4/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://butyr.org:8000/9c9914c2855d0b4/output.txt

Total script time: 0.47 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @secretrobotron received. Current queue size: 0

Live output at: http://butyr.org:8000/51480146cfe9279/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://butyr.org:8000/51480146cfe9279/output.txt

Total script time: 0.66 mins

Server is running on http://pr294.butyr.org/templates/test.html

File: src/core/media.js [hunk 1]
```diff
@@ -124,19 +124,15 @@
...
-        var newTrack = e.target,
-            trackEvent = e.data;
-        _popcornWrapper.updateEvent( trackEvent );
+        _popcornWrapper.updateEvent( e.data );
       } //onTrackEventAdded
 
       function onTrackEventUpdated( e ){
-        var trackEvent = e.target;
-        _popcornWrapper.updateEvent( trackEvent );
+        _popcornWrapper.updateEvent( e.data );
```
Comment (secretrobotron) [inline src/core/media.js]: These aren't the same, but it might still work because e.data === e.target. Check?

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_cleanup** from @jbuck received. Current queue size: 0

Live output at: http://butyr.org:8000/b0c5d5d70c61459/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://butyr.org:8000/b0c5d5d70c61459/output.txt

Total script time: 1.19 mins

**Pull request merged**